### PR TITLE
Improve welcome documentation and add common questions

### PR DESCRIPTION
**Pull request by hillaryj**
#  #

## 🗣 Description ##

Adds common questions for getting started with GitHub, with cisagov, and other resources.

## 💭 Motivation and context ##

Make it easier to figure out how to use cisagov and what to do to get started or do common tasks.

## 🧪 Testing ##

Documentation-only, so linting.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,94 @@
...
 
 [![GitHub Build Status](https://github.com/cisagov/cisagov/workflows/build/badge.svg)](https://github.com/cisagov/cisagov/actions)
 
-Welcome to `cisagov`, the GitHub organization for the Cybersecurity and
-Infrastructure Security Agency (CISA).
+Welcome to `cisagov`, the GitHub home for the Cybersecurity and Infrastructure
+Security Agency (CISA)!
 
-## Getting started with GitHub ##
+This repository is aimed to make it easier to get working with GitHub and Free
```
Comment (jsf9k) [inline README.md]:
```suggestion
This repository aims to make it easier to get working with GitHub and Free
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,94 @@
...
+account?
+  - GitHub's terms of service say to use one account per person.
+  Any commits made will be associated with the user who created them, and
+  GitHub allows for granular, role-based access that can also be revoked
+  when someone departs CISA.
+- How do I use GitHub? Where do I start?
+  - [18F Intro to GitHub](https://handbook.tts.gsa.gov/intro-to-github/)
+  - [Digital.gov intro to GitHub](https://digital.gov/resources/an-introduction-github/)
+  - Always remember that since GitHub is a version control system, all changes
+  are saved, so there's nothing we can't fix or undo
```
Comment (jsf9k) [inline README.md]:
```suggestion
  - Always remember that GitHub is a version control system, so all changes
  are saved, so there is nothing we can't fix or undo.
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,94 @@
...
+  - For contractors, ask your fed lead to send in your username
+- How do I get put on a team within the `cisagov` organization?
+  - See [Github's organizations and teams documentation](https://docs.github.com/en/organizations)
+  - You'll need to contact a team member or a `cisagov` administrator
+- How do I get access to a specific repository?
+  - Most repositories are associated with the [teams](https://docs.github.com/en/organizations/organizing-members-into-teams)
+  that use them, so to get access you can contact github@cisa.gov, a
+  team member, or a `cisagov` administrator to get access
+- How do I make a new repository?
+  - [GitHub's creating a repository documentation](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository)
```
Comment (jsf9k) [inline README.md]:
```suggestion
  - [GitHub's documentation on creating a repository](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository)
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,94 @@
...
+    - Is the project still under active development or is it in maintenance?
+      - If the project no longer has a team performing maintenance, we
+      recommend the repository be archived to make that clear to people who
+      may want to use it
+- What belongs on [cisa.gov](https://www.cisa.gov) versus on `cisagov`?
+  - The [cisa.gov](https://www.cisa.gov) site is primarily focused at an
+  audience outside of CISA, such as Critical Infrastructure partners or the
+  public
+  - `cisagov` is for both internal and external users and partners and
+  specifically to share projects with the public as well as internal users
```
Comment (jsf9k) [inline README.md]:
```suggestion
  - `cisagov` is for both internal and external users, as well as partners. It exists
  specifically to share projects with the public as well as internal users.
```

File: README.md [hunk 5]
```diff
@@ -1,16 +1,94 @@
...
+- Working in the public (dos and don'ts, best practices)
+  - As a best practice, use the `cisagov` organization issue templates and
+  pull request templates. These templates are available in all repositories
+  created in the `cisagov` organization by default.
```
Comment (jsf9k) [inline README.md]:
```suggestion
  pull request templates. These templates are available by default in 
  all repositories created in the `cisagov` organization.
```

File: README.md [hunk 6]
```diff
@@ -1,16 +1,94 @@
...
+  - As a best practice, we require code reviews before merging pull requests
+  through using [branch protection](https://docs.github.com/en/github/administering-a-repository/about-protected-branches)
```
Comment (jsf9k) [inline README.md]:
```suggestion
  - As a best practice, we require code reviews before merging pull requests.
  This is done using [branch protection](https://docs.github.com/en/github/administering-a-repository/about-protected-branches).
```

**Review by jsf9k — APPROVED** [review]
Approved.  I made a few suggestions that I think improve readability, but feel free to ignore them if you disagree.

File: README.md [hunk 7]
```diff
@@ -1,16 +1,94 @@
...
+account?
+  - GitHub's terms of service say to use one account per person.
+  Any commits made will be associated with the user who created them, and
+  GitHub allows for granular, role-based access that can also be revoked
+  when someone departs CISA.
+- How do I use GitHub? Where do I start?
+  - [18F Intro to GitHub](https://handbook.tts.gsa.gov/intro-to-github/)
+  - [Digital.gov intro to GitHub](https://digital.gov/resources/an-introduction-github/)
+  - Always remember that GitHub is a version control system, so all changes
+  are saved, so there is nothing we can't fix or undo.
```
Comment (dav3r) [inline README.md]:
```suggestion
  are saved and there is nothing we can't fix or undo.
```

File: README.md [hunk 8]
```diff
@@ -1,16 +1,94 @@
...
+      - If the project no longer has a team performing maintenance, we
+      recommend the repository be archived to make that clear to people who
+      may want to use it
+- What belongs on [cisa.gov](https://www.cisa.gov) versus on `cisagov`?
+  - The [cisa.gov](https://www.cisa.gov) site is primarily focused at an
+  audience outside of CISA, such as Critical Infrastructure partners or the
+  public
+  - `cisagov` is for both internal and external users, as well as partners. It exists
+  specifically to share projects with the public as well as internal users.
+- Working in the public (dos and don'ts, best practices)
```
Comment (dav3r) [inline README.md]: Take or leave this one, but to my eyes, it works better this way:
```suggestion
- Working in public (dos and don'ts, best practices)
```

File: README.md [hunk 9]
```diff
@@ -1,16 +1,94 @@
...
+  - Early and often!
+- What is CISA's open source policy?
+  - See [CISA's open source policy here](https://github.com/cisagov/development-guide/blob/develop/open-source-policy/policy.md)
+
+## Feedback and contact ##
+
+Have an idea about how to make these pages better? [File an issue!](https://github.com/cisagov/cisagov/issues/new/choose)
+
+For any repository-specific questions or feedback, please make an issue in
+that repository so the appropriate team will see it.
```
Comment (dav3r) [inline README.md:86]: Should we also mention here that some of our repositories have [GitHub Discussions](https://docs.github.com/en/discussions) enabled and that is another avenue for questions?

**Review by dav3r — APPROVED** [review]
Looks great! 💎 
I left two small suggestions and a question, so please take a look at those.

File: README.md (hunk 8 above)
Comment (hillaryj) [reply README.md]: Looks great!

File: README.md (hunk 9 above)
Comment (hillaryj) [reply README.md:86]: The issue-chooser mentions Discussions and Wiki, plus links to this: https://github.com/cisagov/.github/blob/develop/QUESTIONS.md

I'm ambivalent, slightly on the side of leaving it simpler here in the general suggestion. This way the path is obviously "have a question" -> "go to the repository".

Comment (dav3r) [reply README.md:86]: Good point- I forgot that we mention it in the issue-chooser, so I'm on board with keeping it simple.

**Pull request merged**